components/train.yaml

```diff
@@ -106,6 +108,22 @@ implementation:
           ${bootleg_args} \
           $@
 
+      if test "$calibrate" == "true" ; then
+        genienlp predict \
```

Comment:
One part I'm not sure is why do we need a separate predict step? `genienlp train` already evaluates on the validation set, it can compute the confidence features as well.

Reply:
On the other hand, given https://github.com/stanford-oval/genienlp/pull/72#issuecomment-758174725, should we then pass a different validation set for calibration, one that has a lot of OOD stuff?
Note that (a) I don't want the OOD to affect how I visualize/analyze in-domain performance, and (b) I want to store the OOD in S3 not in github like the rest of the eval set because it contains a lot of questionable stuff.